### ISSUE TITLE
Step Overview : Step Frequency

### ISSUE THREAD

**Issue by gazler22** [trivago/cluecumber-report-plugin#147]
Ok, so I took the new release and its wonderful! Thank You :)

I do have one request though about the step overview page. I am not sure if I should have reopened the previous one (#145 ), so please let me know and I ll update this.

The current page shows the **passed**, **failed** and **skipped** frequencies along with the steps.

But these are actually the **counts** of the step being used in scenarios (which failed, passed or skipped) and not the frequency of the steps having **passed**, **failed** and **skipped** themselves.

I think it would be a lot more useful if we instead show the actual data of the steps themselves having failed or not, instead of the number of times they are used in a scenario.

This helps to get a better perspective in case of TestSuits with reused steps in a scenario.

For eg: 
```
Scenario Outline : Get total fruits left

Given I have fruits

And I have 10 "apple"
When I eat 2 of them
Then I am left with 8 of them

Given I have 3 "banana"
When I eat none
Then I am left with 3 of them

And I have 10 total fruits left
```

If I have multiple features with similar scenario outlines like this one (maybe different steps in them along with these), and **all of these features fail at the 'I have fruits' step**, the step overview page will **still** show **'I have {} {}'** at the top if I sort the steps by the **failed** tab in the step overview page (since its the most used in failed scenarios even though its not the most failed one).

But it would be more useful if it showed them by their individual failure frequency to know at a glance that the 'I have fruits' step has failed for all the scenarios when I sort by failed.

Or maybe we could add another tab to show the frequency of **execution** of these steps along with the current one which shows the frequency of the **presence** of these steps in scenarios.

Please let me know if I wasn't clear and maybe I can make a sample project to clarify what I mean :)

**Comment by laxersaz** [comment]
Yes I get what you mean and I see the benefits of seeing the individual step results. This is more interesting than their connected scenario state. So I will change that for the next release.

**Comment by gazler22** [comment]
Thank you :)

Let me know if u want me to start/assist with this, it's the weekend after all :)

**Comment by laxersaz** [comment]
This is fixed in 1.8.1

**Comment by gazler22** [comment]
@laxersaz  Thank you this was super helpful :)
